src/js/cli-engine.js

```diff
@@ -139,14 +139,21 @@ class CLIEngine {
             return null;
         }
 
-        const formatterPath = `./formatters/${format}`;
```

Comment:
What is the purpose of this change?

Reply:
This happened around the period we were experimenting with Webpack. I ended up hardcoding the paths in the require statement to get it to load properly. I will revert.